shell/plugins/steve/steve-class.js

```diff
@@ -2,6 +2,19 @@ import { DESCRIPTION } from '@shell/config/labels-annotations';
 import HybridModel from './hybrid-class';
 
 export default class SteveModel extends HybridModel {
+  constructor(data, ctx, rehydrateNamespace = null, setClone = false) {
+    let _description;
```

Comment:
Haven't looked in to this, but double check SteveDescriptionModel, it might do exactly this